src/env.zig

```diff
@@ -17,7 +17,7 @@ pub const isMac = build_target == .native and @import("builtin").target.os.tag =
 pub const isBrowser = !isWasi and isWasm;
 pub const isWindows = @import("builtin").target.os.tag == .windows;
 pub const isPosix = !isWindows and !isWasm;
-pub const isDebug = std.builtin.Mode.Debug == @import("builtin").mode;
+pub const isDebug = false;
```

Comment:
That was an accident, I'll revert this

Reply:
Here is a little nudge
```diff
- pub const isDebug = false;
+ pub const isDebug = std.builtin.Mode.Debug == @import("builtin").mode;
```
Just accept the changes 👇